components/brave_wallet_ui/components/extension/connected-panel/style.ts

```diff
@@ -165,3 +165,8 @@ export const MoreAssetsButton = styled(WalletButton)`
   letter-spacing: 0.01em;
   color: ${(p) => p.theme.palette.white};
 `
+
+export const Spacer = styled.div`
```

Comment:
How about renaming this as `VerticalSpacer` & adding it to the shared styles with a `space` prop? That way we can reuse it

Reply:
Done!